src/Features/CSharp/Portable/Structure/CSharpStructureHelpers.cs

```diff
@@ -275,9 +275,9 @@ static bool TryGetLeadingCollapsibleSpan(SyntaxNode node, out BlockSpan span)
                     return false;
                 }
 
-                var firstComment = startToken.LeadingTrivia.FirstOrNull(t => t.IsKind(SyntaxKind.SingleLineCommentTrivia));
+                var firstComment = startToken.LeadingTrivia.FirstOrNull(t => t.IsKind(SyntaxKind.SingleLineCommentTrivia) || t.IsKind(SyntaxKind.SingleLineDocumentationCommentTrivia));
```

Comment:
❓why do we not handle multiline comments the same here?

Reply:
It doesn't seem that way, but this behaviour is only for documentation comments, its just that in metadata as source, documentation comments get re-written as regular comments. Why? No idea! @sharwell probably knows

This fix is because we now will GTD for embedded source, the re-writing doesn't happen so we're showing real documentation comments, and they weren't being handled, ironically enough.